### ISSUE TITLE
Post draft comment with file path and line number

### ISSUE THREAD

**Issue by kumar303** [mozilla/addons-server#11378]
### Describe the problem and steps to reproduce it:

An imaginary UI flow for commenting might look like this:

- As a reviewer, click the line of a diff to enter a comment
- Type a comment, press save

### What happened?

n/a

### What did you expect to happen?

The comment should be saved by the backend

### Anything else we should know?

I don't see a POST endpoint in [the comments API](https://addons-server.readthedocs.io/en/latest/topics/api/reviewers.html#drafting-comments) so we need that for starters.

We also need to be able to post all data that will be needed for https://github.com/mozilla/addons-server/issues/11374 , such as file path, line number, and anything else.

**Comment by AlexandraMoga** [comment]
@EnTeQuAk I'm running into some errors with the [POST draft-comments](https://addons-server.readthedocs.io/en/latest/topics/api/reviewers.html#post--api-v4-reviewers-addon-(int-addon_id)-versions-(int-version_id)-draft_comments-) endpoint

To eliminate other possible error sources, I've made sure that the add-on I'm requesting is valid and my auth signature is correct.
Then, I've sent the following request:
```
POST https://addons-dev.allizom.org/api/v4/reviewers/addon/385621/versions/1655456/draft_comments/

Body:
{"comment": "My first comment"}
```
When I'm providing a valid string for `comment`, I receive a 500 server error.

What I found interesting is that this API does validate an empty string for `comment`, namely the following request will respond with a 400:
```
Request body:
{"comment": ""}

Response body:
{    "comment": [
        "This field may not be blank."
    ]
}

**Comment by EnTeQuAk** [comment]
This seems to be related to https://sentry.prod.mozaws.net/operations/olympia-dev/issues/6054743/ - and a migration that hasn't been run properly. I'll fix this on -dev.

**Comment by EnTeQuAk** [comment]
I fixed the migration on -dev manually, we had some weird double migrations lately so something may have gone wrong.

**Comment by AlexandraMoga** [comment]
I've verified this on -dev with the [POST draft_comment](https://addons-server.readthedocs.io/en/latest/topics/api/reviewers.html#post--api-v4-reviewers-addon-(int-addon_id)-versions-(int-version_id)-draft_comments-) endpoint.

Draft comments are stored according to the parameters sent. There is no validation done for `filename` and `lineno`, meaning that that any value could go into these fields right now, provided they are the correct type (string/int). 

I've also verified that the api responds correctly in case the user doesn't have the necessary permissions to access it.
